Bottom: 8604aab77dbb6ee33163b68b324b9ccda51d7a38
Top:    c37c193a8dd7b2fe4234ee818aaec78b5b6f4b9f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:43:44 +0000

Fix for arc bounding box calculation split from later patch

This probably wants merging with where the problem was introduced,
but at this moment I've not had chance to review the fix is good.


---

diff --git a/src/misc.c b/src/misc.c
index e0bdb4a..678ec6a 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -1603,10 +1603,10 @@ calc_thin_arc_bounds (Coord cx, Coord cy, Coord rx, Coord ry, Angle start_angle,
   sa2 = sin (M180 * ang2);
   ca2 = cos (M180 * ang2);
 
-  bound.X1 = MIN (ca1, ca2);
-  bound.X2 = MAX (ca1, ca2);
-  bound.Y1 = MIN (sa1, sa2);
-  bound.Y2 = MAX (sa1, sa2);
+  minx = MIN (ca1, ca2);
+  maxx = MAX (ca1, ca2);
+  miny = MIN (sa1, sa2);
+  maxy = MAX (sa1, sa2);
 
   /* Check for extreme angles */
   if ((ang1 <= 0   && ang2 >= 0)   || (ang1 <= 360 && ang2 >= 360)) maxx = 1;
